service/templates/projects.html

```diff
@@ -19,13 +19,13 @@
 Evergreen Projects
 {{end}}
 {{define "content"}}
-<div id="content" class="container-fluid row" ng-controller="ProjectCtrl" ng-hide="isAdmin">
+<div id="content" class="container-fluid row" ng-controller="ProjectCtrl" ng-hide="projectView">
```

Comment:
Subtle difference, but since scope.projectView is set a split second after the page loads, hiding it this way will show this section for a split second. If you invert the check by doing ng-show="!projectView" I think that will prevent it

Reply:
thank you!

Reply:
It still flashes for a second, even with ng-show="!projectView"